### PR TITLE
Cmdline Args for UP Server

### PR DESCRIPTION
**Pull request by franklinselva**
Added command line arguments for
- server address
- binary problem file

### CODE REVIEW

File: model/src/lang/reification.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::type_complexity)] // TODO: This is a temporary solution to avoid the warning. Fix type complexity.
```
Comment (arbimo) [inline model/src/lang/reification.rs:1]: Not sure it is still required. Even if it is, it should be made more targeted (i.e. not file wide).

Comment (franklinselva) [reply model/src/lang/reification.rs]: Yes, I have reverted the changes.

**Comment by arbimo** [comment]
This seems to be quite ad-hoc: we expect one or two arguments, which are essentially mutually exclusive (if there is a file on the command line, the address is unused). It is also non discoverable and not documented. A better way would be to use clap (https://docs.rs/clap/latest/clap/) to handle the argument parsing (in particular, I think subcommands would make sense here where we have two exculsive cases).

**Comment by franklinselva** [comment]
@arbimo I have added the requested changes.

**Review by arbimo — APPROVED** [review]

**Pull request merged**